testcases/MachineConfig.py

```diff
@@ -207,6 +211,61 @@ def runTest(self):
             self.fail("Failed to boot in %s mode" % proc_mode)
 
 
+class OsConfig(MachineConfig):
+    '''
+    This Class assign  huge page in the system   based on MMU either Radix or hash and validate
+    MMU is Radix : 2M or 1 GB Huge page
+    MMU HASH :  16M
+    pass hgpgsize to os_config in config file
+    '''
+    def setUp(self):
+        super(OsConfig, self).setUp()
+        conf = OpTestConfiguration.conf
+        self.os_level = self.cv_HOST.host_get_OS_Level()
+        try: self.os_config = conf.args.os_config
+        except AttributeError:
+            self.os_config = "hppgsize=1G"
```

Comment:
lets not have a new parameter os_config,, instead we will be passing hugepage parameter to existing op-test param machine-config

Reply:
okay we will pass it to machine_config itself